### PR TITLE
simgrid 3.17

### PR DESCRIPTION
**Pull request by aik8**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/simgrid.rb [hunk 1]
```diff
@@ -1,20 +1,11 @@
 class Simgrid < Formula
   desc "Studies behavior of large-scale distributed systems"
   homepage "http://simgrid.gforge.inria.fr"
-  url "https://gforge.inria.fr/frs/download.php/file/33686/SimGrid-3.11.1.tar.gz"
-  sha256 "7796ef6d4288462fdabdf5696c453ea6aabc433a813a384db2950ae26eff7956"
-
-  bottle do
```
Comment (ilovezfs) [inline Formula/simgrid.rb]: The bottle block should not be changed or removed. It's updated programmatically.

**Comment by ilovezfs** [comment]
looks like it needs a `:python3` dep

File: Formula/simgrid.rb (hunk 1 above)
Comment (ProteusCortex) [reply Formula/simgrid.rb]: Oh. Alright. Will fix that.

**Comment by ProteusCortex** [comment]
Yeah, just checked it out. Will fix that too.

File: Formula/simgrid.rb [hunk 2]
```diff
@@ -15,6 +15,8 @@ class Simgrid < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "python3" => :build
```
Comment (ilovezfs) [inline Formula/simgrid.rb]: are you sure this is a build-time only requirement? That is very rare.

Comment (ProteusCortex) [reply Formula/simgrid.rb]: Sorry. My mistake. Copied the whole line.

File: Formula/simgrid.rb [hunk 3]
```diff
@@ -15,6 +15,8 @@ class Simgrid < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "python3"
```
Comment (DomT4) [inline Formula/simgrid.rb]: `:python3` not `"python3"`.

**Comment by ilovezfs** [comment]
Thanks @ProteusCortex!

**Pull request closed without merging**